src/app/communities/communityUtils.tsx

```diff
@@ -203,6 +210,7 @@ export const rootstockHacktivator: CommunityItem = {
   cover: '',
   detailedDescription: <></>,
   specialPower: '',
+  activation: '',
   requirement: '',
```

Comment:
Tech debt here, these should be optional parameters not blank, but given this is more of a 'hotfix' I'd suggest we defer this.